spec/features/user_can_delete_station_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../spec_helper'
+require_relative 'load_database'
+
+include LoadDatabase
+
+describe 'user deletes station' do
+  it 'can be deleted from show page' do
+    LoadDatabase.load
+    visit('/stations/2')
+    click_on('Delete')
+
+    page.has_xpath?('/stations')
```

Comment:
I don't understand this xpath. What exactly is this asserting? I think there's a simpler assertion we can use here because xpath seems to have a lot of functionality we're not using. Maybe something like expect(page).to have_current_path